common/src/commonMain/kotlin/com/trikot/sample/viewmodels/home/HomeViewModelController.kt

```diff
@@ -0,0 +1,19 @@
+package com.trikot.sample.viewmodels.home
+
+import com.mirego.trikot.kword.I18N
+import com.mirego.trikot.streams.reactive.RefreshablePublisher
+import com.trikot.sample.domain.FetchQuoteUseCase
+import com.trikot.sample.viewmodels.base.AbstractViewModelController
+import com.trikot.sample.viewmodels.base.BaseNavigationDelegate
+import com.trikot.sample.viewmodels.home.impl.HomeViewModelImpl
+
+class HomeViewModelController(
+    fetchQuoteUseCase: FetchQuoteUseCase,
+    i18N: I18N
+) : AbstractViewModelController<BaseNavigationDelegate, HomeViewModel>() {
+    private val refreshableQuotePublisher = RefreshablePublisher({ _, _ ->
+        fetchQuoteUseCase.fetchQuote()
+    })
+
+    override val viewModel: HomeViewModel = HomeViewModelImpl(refreshableQuotePublisher, i18N)
```

Comment:
wondering if we should encourage the `by lazy` pattern here (cc @npresseault )

Reply:
I would not at all! The lazy part should be related to the subscription to the publisher, not the creation of the objet. Moreover, if you have a publisher subscribed before the lazy, it might freeze you object and crash.

Reply:
Good point !